### PR TITLE
e2e: move funs of framework/metrics to e2e_node

### PR DESCRIPTION
**Pull request by haosdent**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Move funs of framework/metrics to e2e_node

**Which issue(s) this PR fixes**:

Ref #86052 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/e2e_node/density_test.go [hunk 1]
```diff
@@ -20,6 +20,8 @@ package e2enode
 
 import (
 	"fmt"
+	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
```
Comment (oomichi) [inline test/e2e_node/density_test.go]: The PR job is failed like:
```
Execution platform: //:rbe_with_network
compilepkg: missing strict dependencies:
	/b/f/w/test/e2e_node/density_test.go: import of "k8s.io/kubernetes/test/e2e/framework/log"
No dependencies were provided.
Check that imports in Go sources match importpath attributes in deps.
```
It is necessary to run `./hack/update-bazel.sh` and apply the change into this PR.

In addition, this line(e2elog) should be moved to line 41 which is the same block of e2e sub framework.

**Review by oomichi — CHANGES_REQUESTED** [review]

File: test/e2e_node/density_test.go (hunk 1 above)
Comment (haosdent) [reply test/e2e_node/density_test.go]: fixed, thanks @oomichi could you help to review again? Thank you in advance.

**Comment by oomichi** [comment]
Cool, thanks for doing this.

/lgtm

**Comment by oomichi** [comment]
I heard `[sig-network] Services should have session affinity work for service with type clusterIP [LinuxOnly]` can be resolved.

/retest

**Pull request merged**